### DISCUSSION TITLE
Push on github can't show background

### DISCUSSION THREAD

**Discussion by KenHuangTW** [hexojs/hexo#5051] [Question / Answer] [answered]
Chinese:
我在 main.styl 裡面設定背景圖片
```
body {
    background:url(/images/background.jpg);
    background-repeat: no-repeat;
    background-attachment:fixed;
    background-position:50% 50%;
}
```
並且在 localhost確認是可以顯示的
但一推上 github 就沒辦法顯示背景圖片了，請問我有少做哪個必要步驟嗎?
除了
```
hexo clean
hexo g
hexo d
```

English:
I set background in path: blog\themes\next\source\css\main.styl

```
body {
    background:url(/images/background.jpg);
    background-repeat: no-repeat;
    background-attachment:fixed;
    background-position:50% 50%;
}
```
Than I use `hexo server` to check on localhost:4000 , it's normal.
But when I push on github, background always can't show.

**Comment by KenHuangTW** [comment, marked as answer]
And You just wait sometimes.

**Comment by yoshinorin** [comment]
@KenHuangTW 

Is this problem solved?
I visit your site and I can see the background image correctly.

> https://kenhuangtw.github.io/

**Reply by KenHuangTW** [reply]
Yes. That can work on github page.  Just wait sometimes.
